t/plugin/error-log-skywalking-logger.t

```diff
@@ -0,0 +1,357 @@
+#
```

Comment:
Should use `t/plugin/error-log-logger-skywalking.t`, the prefix is the plugin name.